images/prow-tests/Dockerfile

```diff
@@ -15,9 +15,12 @@
 FROM gcr.io/k8s-testimages/kubekins-e2e:v20210113-cc576af-master AS stable-stuff
 
 ENV DEBIAN_FRONTEND noninteractive
+ARG COMMIT_HASH
 ARG GCLOUD_VERSION=368.0.0
 ARG KUBECTL_VERSION=v1.21.4
 
+RUN echo "${COMMIT_HASH}" > /commit_hash
```

Comment:
I guess it's already the plan but just want to double check - after this PR is merged, are you going to add one line in https://github.com/knative/hack/blob/main/presubmit-tests.sh#L252-L283 to print out the commit hash?

Reply:
that is right